webapp/portlet/src/main/webapp/vue-apps/getting-started/main.js

```diff
@@ -12,6 +12,8 @@ export function hideGettingStarted() {
 const parentAppElement = document.querySelector('#GettingStartedPortlet .btClose');
 if (parentAppElement) {
   parentAppElement.onclick = () => {
-    hideGettingStarted().then(() => document.querySelector('#GettingStartedPortlet').parentElement.remove());
+    hideGettingStarted().then(() => document.querySelector('#GettingStartedPortlet').parentElement.parentElement.remove());
   };
+} else {
+  document.querySelector('#GettingStartedContainerChildren .PORTLET-FRAGMENT').classList.add('hidden');
```

Comment:
```suggestion
} else {
  const parentElementToHide = document.querySelector('#GettingStartedPortlet');
  if (parentElementToHide) {
    parentElementToHide.closest('.PORTLET-FRAGMENT').classList.add('hidden');
  }
```

Reply:
WIth this suggestion we will have a js error (element with id GettingStartedPortlet not found) because we delete this element  here 
https://github.com/Meeds-io/social/blob/7834dc383d70a8a7dae319a8a2ad78527aaf42a3/webapp/portlet/src/main/webapp/vue-apps/getting-started/main.js#L15
that's why  I use this selector
document.querySelector('#GettingStartedContainerChildren .PORTLET-FRAGMENT')

Reply:
done